jeos-firstboot-enroll

```diff
@@ -264,6 +283,12 @@ enroll_tpm_and_fido() {
 			enroll_fido2 "$dev"
 		done
 	elif [ -n "$with_tpm2" ]; then
+		if [ -n "$crypt_tpm_pin" ]; then
+			# XXX ./src/cryptenroll/cryptenroll-tpm2.c lacks accept cached
+			#echo -n "$crypt_tpm_pin" | run keyctl padd user tpm2-pin @u
+			export NEWPIN="$crypt_tpm_pin"
```

Comment:
Do not export the secret:
https://github.com/openSUSE/disk-encryption-tool/pull/12/files#diff-b3ccc79ad196422dc257215e20dc1d7a01b08b3e4d2ba43850689f15804042f3R134